include/rdma/fi_eq.h

```diff
@@ -76,8 +78,13 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
+#ifndef _WIN32
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
+#else /* _WIN32 */
+	CRITICAL_SECTION	*mutex;
+	CONDITION_VARIABLE	*cond;
+#endif /* _WIN32 */
```

Comment:
To be clear, I'm suggesting that this change to:

```c
#ifndef _WIN32
struct fi_mutex_cond {
...
};
#else 
struct fi_critsec_cond {
...
};
#endif
```

This way the critical section implementation is exposed as a critical section, rather than trying to incorrectly expose it as a mutex.